server/app/controllers/barman-controller.js

```diff
@@ -108,10 +109,77 @@ async function deleteBarman(req, res) {
 
 }
 
+/**
+ * Get the services of a barman
+ *
+ * @param req Request
+ * @param res Response
+ * @returns {Promise<void>} Nothing
+ */
+async function getServicesBarman(req, res) {
+    const barmanId = req.params.id;
+
+    const services = await barmanService.getBarmanServices(barmanId);
+
+    res.json(services);
+}
+
+/**
+* Create a Service
+*
+* @param req Request
+* @param res Response
+* @return {Promise.<void>} Nothing
+*/
+async function createServiceBarman(req, res) {
+    const barmanId = req.params.id;
+    const serviceId = req.params.idService;
+
+    if (!checkStructure(req.body, ['startAt', 'endAt', 'nbMax', 'category'])) {
```

Comment:
This path take an array of services (for simplicity)

Reply:
Actually, there was some mistakes with the API specs, it's updated and now there is just an array of ids